pkg/command/service/scale.go

```diff
@@ -189,7 +189,7 @@ func runScaleFromZero(ctx context.Context, params *pkg.PerfParams, inputs pkg.Sc
 	sdch := make(chan struct{})
 	errch := make(chan error)
 
-	endpoint, err := resolveEndpoint(ctx, params, inputs.ResolvableDomain, svc)
+	endpoint, err := resolveEndpoint(ctx, params, inputs.ResolvableDomain, false, svc)
```

Comment:
just to check: should this be `inputs.Https` instead of `false` or do we always want this to be false here?

Reply:
It is `false` for `scale` by default, will add `HTTPS` support in `scale` for update.

Reply:
To make code easier to read, I also moved functions used by both load and scale to common.